fdbserver/RestoreMaster.actor.cpp

```diff
@@ -204,9 +204,13 @@ ACTOR Future<Void> startProcessRestoreRequests(Reference<RestoreMasterData> self
 	} catch (Error& e) {
 		if (restoreIndex < restoreRequests.size()) {
 			TraceEvent(SevError, "FastRestoreMasterProcessRestoreRequestsFailed", self->id())
+			    .detail("Error", e.what())
+			    .detail("ErrorCode", e.code())
```

Comment:
This and the one below can be simplified as
```suggestion
			    .error(e)
```